OpenRA.Mods.Common/Traits/Buildings/BuildingUtils.cs

```diff
@@ -17,7 +17,7 @@ namespace OpenRA.Mods.Common.Traits
 {
 	public static class BuildingUtils
 	{
-		public static bool IsCellBuildable(this World world, CPos cell, BuildingInfo bi, Actor toIgnore = null)
+		public static bool IsCellBuildable(this World world, CPos cell, ActorInfo ai, BuildingInfo bi, Actor toIgnore = null)
```

Comment:
What is this change for? Just consistency with `CanPlaceBuilding`?

Reply:
Indeed.